driverthroughput/README.md

```diff
@@ -0,0 +1,253 @@
+    --wall_time=WALL_TIME
+        Default: '05:00:00'
+        allotted time for database launcher to run
+    --plot=PLOT
+        Default: 'database_nodes'
+        flag to plot against in process results
+```
+
+The standard throughput tests will spin up a database for each iteration in the
+battery of tests chosen by the user. There are multiple ways to run this.
+
+1. Everything in the same interactive (or batch file) without caring about placement
+```bash
+# alloc must contain at least 60 (max client_nodes) + 32 nodes (max db_nodes)
+python driver.py throughput_standard --client_nodes=[60] \
+                                    --clients_per_node=[48] \
+                                    --db_nodes=[32] \
+                                    --db_cpus=[32] --net_ifname="ipogif0" \
+                                    --run_db_as_batch=False
+```
+
+This option is recommended as it's easy to launch in interactive allocations and
+as a batch submission, but if you need to specify separate hosts for the database
+you can look into the following two methods.
+
+A batch submission for this first option would look like the following for Slurm
+based systems.
+
+```bash
+#!/bin/bash
+
+#SBATCH -N 92
+#SBATCH --exclusive
+#SBATCH -t 10:00:00
+#SBATCH -C SK48
+#SBATCH --oversubscribe
+
+cd ..
+python driver.py throughput_standard --client_nodes=[60] \
+                                    --clients_per_node=[48] \
+                                    --db_nodes=[32] \
+                                    --db_cpus=[32] --net_ifname="ipogif0" \
+                                    --run_db_as_batch=False
+```
+
+2. Same as 1, but specify hosts for the database
+```bash
+# alloc must contain at least 60 (max client_nodes) + 32 nodes (max db_nodes)
+# db nodes must be fixed if hostlist is specified
+python driver.py throughput_standard --client_nodes=[60] \
+                                    --clients_per_node=[48] \
+                                    --db_nodes=[32] \
+                                    --db_cpus=[32] --net_ifname="ipogif0" \
+                                    --run_db_as_batch=False \
+                                    --db_hosts=["nid0001", ...]
+
+```
+
+3. Launch database as a separate batch submission each time
+```bash
+# must obtain separate allocation for client driver through interactive or batch submission
+# if batch submission, compute nodes must have access to slurm
+python driver.py throughput_standard --client_nodes=[60] \
+                                    --clients_per_node=[48] \
+                                    --db_nodes=[32] \
+                                    --db_cpus=[32] --net_ifname="ipogif0" \
+                                    --run_db_as_batch=True \
+                                    --db_node_feature='{"C":"V100", "exclusive": None}' \
+```
+
+All three options will conduct ``n`` scaling tests where ``n`` is the multiple of
```

Comment:
"multiple" should be "product". (check to see if the same error is present for other files)